src/main/java/sirius/biz/tenants/mongo/MongoTenantAware.java

```diff
@@ -11,14 +11,23 @@
 import sirius.biz.mongo.MongoBizEntity;
 import sirius.biz.tenants.Tenant;
 import sirius.biz.web.TenantAware;
+import sirius.db.mixing.annotations.Index;
+import sirius.db.mongo.Mango;
 import sirius.db.mongo.types.MongoRef;
 import sirius.kernel.di.std.Part;
 
 import java.util.Optional;
 
 /**
  * Base class which marks subclasses as aware of their tenant they belong to.
+ * <p>
+ * Note that an index is automatically created containing the tenant itself and the searchPrefixes,
+ * which are added via {@link MongoBizEntity}. You can skip the index creation by defining an {@link Index}
+ * without columns.
  */
+@Index(name = "tenant_searchPrefixes",
```

Comment:
shouldn't this be named prefix_index to overwrite the index defined by PrefixSearchable Entity?

otherwise we end up with two entities where the original one will almost never be used(?)

Reply:
The real index name is `searchPrefixes_1` since it wasn't using the name property of the annotation. All indexes must be dropped and recreated anyway, therefore I would include the word **tenant** in the name to better indicate how the index is build up.

Reply:
i meant name it so that the parent index as defined by PrefixSearchableEntity is suppressed - although a misleading name in mongo would be created then - still better than a mostly unused index